test/dropbox/openapi-mock.yaml

```diff
@@ -0,0 +1,25 @@
+# OpenAPI specification options
+openapi:
+  specification_url: '/etc/openapi/user_v2.yaml'
```

Comment:
If a configuration value for this tool is given both in the environment, and in the config file, what takes precedence? Maybe it'd be good not to define anything twice so this is easier to debug in the future.